### ISSUE TITLE
Ranger runs in background even after closing terminal

### ISSUE THREAD

**Issue by angadsharma1016** [ranger/ranger#1534]
## System details
Ubuntu 16.04 LTS

###  What I did

1. Ran ranger on the command line
2. Closed the terminal by clicking cross

### Expected behavior

Closing the terminal should kill the attached ranger process 

### Actual behavior

Ranger kept running in the background

**Comment by toonn** [comment]
I doubt this is on ranger. Your terminal is somehow not killing child processes. Are you sure this doesn't happen with other similar programs? An irc client or vim or mc?

**Comment by angadsharma1016** [comment]
Yes I am sure. Did the same with mc, vim and a couple of other processes. Had no trouble with them

**Comment by toonn** [comment]
I'm not sure how to debug this. I can't reproduce it. Any ideas, @Vifon?

**Comment by toonn** [comment]
I think #1367 is related and there's a fix waiting to be reviewed in #1368.
